### PR TITLE
Format uk

### PR DESCRIPTION
**Pull request by NeonMariia**

### CODE REVIEW

**Comment by NeonDaniel** [comment]
@NeonMariia I think these are the users you said could review this: @robinhad @VicGrygorchyk . I can't request reviews from non-maintainers, but anyone should be able to leave a review with comments.

If these changes look good to a native speaker, they are already passing unit tests so one of the maintainers here can merge it

**Comment by robinhad** [comment]
Hi @NeonDaniel !
Thanks for adding me to this pull request. I have a dataset with cases for normalization, need some time to prepare it for evaluation and I'll get back to you.

**Comment by NeonDaniel** [comment]
Associated PR for parsing: https://github.com/OpenVoiceOS/ovos-lingua-franca/pull/51

**Comment by JarbasAl** [comment]
as mention in companion PR #51 

we just had a stable release and are back in alpha versions, I am ok with merging this as is and fix any issues in follow up PRs. This is great work and I'd like to get it in the wild!

Only see some debug prints to remove and maybe a rebase, there seems to be some duplicated code too between this and #51

File: lingua_franca/res/text/uk-uk/and.word
Comment (robinhad) [inline lingua_franca/res/text/uk-uk/and.word:2]: `та` could also be used as "and" word in Ukrainian
Ex: `Гензель та Гретель`

File: lingua_franca/res/text/uk-uk/normalize.json
Comment (robinhad) [inline lingua_franca/res/text/uk-uk/normalize.json:13]: should it have `"нуля": "0"` as in `5 градусів нижче нуля`?

File: lingua_franca/res/text/uk-uk/normalize.json
Comment (robinhad) [inline lingua_franca/res/text/uk-uk/normalize.json:22]: and `двох` as well

File: lingua_franca/res/text/uk-uk/yesno.json
Comment (robinhad) [inline lingua_franca/res/text/uk-uk/yesno.json:3]: should contain `та` - popular variant in western Ukraine

File: lingua_franca/res/text/uk-uk/yesno.json
Comment (robinhad) [inline lingua_franca/res/text/uk-uk/yesno.json:13]: `нє` is a popular variant in western Ukraine

File: lingua_franca/res/text/uk-uk/yesno.json
Comment (robinhad) [inline lingua_franca/res/text/uk-uk/yesno.json:25]: could be `можна` depending on context

**Comment by robinhad** [comment]
Hi! Added a couple of comments with suggestion how to improve, overall looks great!

**Review by robinhad — APPROVED** [review]

**Comment by NeonDaniel** [comment]
It looks like there are a few unaddressed comments, but n approving review from @robinhad. Should there be an issue to include those changes later, or are then intentionally not included?

I don't necessarily want to hold the PR for minor improvements but if they're valid, creating an issue will make sure we don't lose track of them.

**Comment by robinhad** [comment]
Hi @NeonDaniel, they are valid, please include them in separate issue as improvements

**Comment by NeonDaniel** [comment]
> Hi @NeonDaniel, they are valid, please include them in separate issue as improvements

Looks like another PR got most of them so I'll open an issue for the last couple before merging this one.

File: lingua_franca/lang/parse_common.py
Comment (emphasize) [inline lingua_franca/lang/parse_common.py]: debug artifact

File: test/unittests/_trial_temp/_trial_marker
Comment (emphasize) [inline test/unittests/_trial_temp/_trial_marker:1]: leftover file

**Review by emphasize — APPROVED** [review]

File: lingua_franca/res/text/uk-uk/and.word
Comment (ChanceNCounter) [reply lingua_franca/res/text/uk-uk/and.word]: This one appears to have been addressed

File: test/unittests/_trial_temp/_trial_marker
Comment (ChanceNCounter) [reply test/unittests/_trial_temp/_trial_marker:1]: that whole directory, on review

File: lingua_franca/res/text/uk-uk/yesno.json
Comment (ChanceNCounter) [reply lingua_franca/res/text/uk-uk/yesno.json:3]: added

**Comment by ChanceNCounter** [comment]
On final review before merging, there only turned out to be one unresolved addition to the resource files, so I just threw that in. I also found one more stray tempfile directory, so I've removed that and re-requested @emphasize's review so as not to self-merge. Should be good to go.

**Review by emphasize — CHANGES_REQUESTED** [review]
All unittests are commented out, i don't think this is wanted

**Review by emphasize — APPROVED** [review]

**Pull request merged**
